src/main/java/org/apache/sling/junit/impl/AnnotationsProcessor.java

```diff
@@ -35,9 +34,11 @@
 public class AnnotationsProcessor implements TestObjectProcessor {
     private Logger log = LoggerFactory.getLogger(getClass());
     private BundleContext bundleContext;
-    
+    private ArrayList<ServiceGetter<? extends Object>> serviceGetters;
```

Comment:
I prefer using interface types for all fields rather than implementations. That makes refactoring to a different type easier and the diff less noisy.